### ISSUE TITLE
releases

### ISSUE THREAD

**Issue by hansvancalster** [ElsLommelen/protocols#1]
Ik vermoed dat we op de protocols website repo ook moeten zorgen dat er releases aangemaakt worden op basis van de tags. De tags zitten er al in, maar ik zie nog geen releases.

**Comment by ElsLommelen** [comment]
Inderdaad, daarvoor moet je gewoon de workflow `release.yml` uit protocolsource overnemen in deze repo.  Voor de testversie had ik dit initieel niet gedaan, omdat ik eerst de workflows zelf op punt wou hebben, maar doe gerust als je wil.

(Wat ook nog moet gebeuren, is de instellingen zodat het met zenodo gekoppeld wordt.  Dit is al gebeurd in protocolsource (met een testversie voor Zenodo als ik me niet vergis), maar moet dus ook in protocols gebeuren.  Deze moeite zou ik precies niet doen in mijn testversies.)
